### PR TITLE
Fix wrong test import and increase BFF client coverage

### PR DESCRIPTION
**Pull request by bbengfort**
### Scope of changes

SC-6346: Fixes an incorrect import in the tests and improves the BFF API definitions test coverage.

### Type of change

- [ ] bug fix
- [ ] new feature
- [ ] documentation
- [x] other (tech debt)

### Acceptance criteria

Describe how reviewers can test this change to be sure that it works correctly. Add a checklist if possible

### Author checklist

- [x] I have manually tested the change and/or added automation in the form of unit tests or integration tests
- [ ]  ~I have updated the dependencies list~
- [ ]  ~I have recompiled and included new protocol buffers to reflect changes I made~
- [ ]  ~I have added new test fixtures as needed to support added tests~
- [x]   Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)
- [x]  I have moved the associated Shortcut story to "Ready for Review"

### Reviewer(s) checklist

- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.

### CODE REVIEW

File: pkg/bff/api/v1/errors_test.go [hunk 1]
```diff
@@ -2,15 +2,49 @@ package api_test
...
+	"errors"
+	"fmt"
 	"net/http"
 	"net/http/httptest"
 	"testing"
 
 	"github.com/gin-gonic/gin"
 	"github.com/stretchr/testify/require"
-	. "github.com/trisacrypto/directory/pkg/gds/admin/v2"
+	. "github.com/trisacrypto/directory/pkg/bff/api/v1"
```
Comment (bbengfort) [inline pkg/bff/api/v1/errors_test.go:13]: This was the oopsie.

File: pkg/bff/api/v1/client_test.go [hunk 2]
```diff
@@ -189,7 +189,7 @@ func TestRegister(t *testing.T) {
 	require.Equal(t, fixture.PKCS12Password, out.PKCS12Password)
 }
 
-func TestVerifyEmail(t *testing.T) {
+func TestVerifyContact(t *testing.T) {
```
Comment (bbengfort) [inline pkg/bff/api/v1/client_test.go:192]: Renaming to the correct RPC name.

File: pkg/bff/api/v1/errors_test.go (hunk 1 above)
Comment (DanielSollis) [reply pkg/bff/api/v1/errors_test.go:13]: oof

**Review by DanielSollis — APPROVED** [review]
Not much to go over, Looks Good!

**Pull request merged**